File: programmers/가운데 글자 가져오기.py
def solution(s):
    answer = ''
    if len(s) % 2 != 0: # 홀수
        answer = s[(len(s) - 1) // 2]
    else: # 짝수
        answer = s[len(s) // 2 - 1:len(s) // 2 + 1]
    return answer

File: programmers/level1/가운데_글자_가져오기.cpp
#include <string>

using namespace std;

string solution(string s) {
    string answer{ "" };
    int mid = s.length() / 2;
    if (s.length() % 2 == 1) {
        answer += s[mid]; // char이므로 바로 return은 불가
    } else {
        answer += s.substr(mid-1,2);
    }
    return answer;
}